content/en/docs/aws/troubleshooting-aws.md

```diff
@@ -132,6 +132,49 @@ ks delete default -c ${COMPONENT}
 ks param set ${COMPONENT} dnsName fs-0xxxxx2a216cf.fsx.us-west-2.amazonaws.com
 ks apply default -c ${COMPONENT}
 ```
+### Amazon RDS Connectivity Issues
+
+If you run into CloudFormation deployment errors, you can use [troubleshooting guide](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/troubleshooting.html) to find a resolution.
+
+If you have connectivity issues with Amazon RDS, try launching mysql-client container and connecting to your RDS endpoint. This will let you know if you have network connectivity with the db and also if db is created properly
+
+```
+# Remember to change your RDS endpoint, DB username and DB Password
+$ kubectl run -it --rm --image=mysql:5.7 --restart=Never mysql-client -- mysql -h <YOUR RDS ENDPOINT> -u admin -pKubefl0w                                                              
```

Comment:
Question: is there a space between -p and Kubefl0w?

Reply:
nope, no space in between. that's how mysql client runs actually : )